plugin/CactbotEventSource/FFXIVProcessKo.cs

```diff
@@ -42,7 +42,7 @@ public unsafe struct EntityMemory {
       [FieldOffset(0xB0)]
       public Single rotation;
 
-      [FieldOffset(0x18A4)]
+      [FieldOffset(0x18B8)]
```

Comment:
Can you update the comment at the top as well for which version this applies to?